.github/workflows/release.yml

```diff
@@ -156,6 +159,15 @@ jobs:
           compression-level: 0
           if-no-files-found: error
 
+      - name: Upload artifact -- linux/amd64/v2
+        uses: actions/upload-artifact@834a144ee995460fba8ed112a2fc961b36a5ec5a  ## v4.3.6
+        with:
+          name: ${{ env.APPLICATION }}_${{ inputs.release_version }}_linux_amd64v2.tar.gz
+          path: ./dist/${{ env.APPLICATION }}_${{ inputs.release_version }}_linux_amd64v2.tar.gz
+          retention-days: 1
```

Comment:
1 retention day only? People should be fast inspecting this artifact ;)

Reply:
For dry-run mode it should be enough. Such artifacts could be used only by Erigon team. I do not see any sense to store it longer.
Otherwise workflow could be executed once again and new artifacts could be created.